hack/test-e2e-local.sh

```diff
@@ -57,7 +57,8 @@ if [[ "$TYPE" == "operator" ]] || [[ "$TYPE" == "operator-seed" ]]; then
   fi
 
   if [[ "$TYPE" == "operator-seed" ]]; then
-    # /etc/hosts must be updated before garden can be created
+    # /etc/hosts must have been updated before garden can be created (otherwise, we could put this command to the
+    # hack/ci-e2e-kind-operator-seed.sh script.
     echo "> Deploying Garden and Soil"
     make operator-seed-up
   fi
```

Comment:
`make operator-seed-up` creates the `Garden` if I understand it correctly. However, `/etc/hosts` is updated below. Therefore, I am still confused by the comment.
(In addition to that, there is a closing parenthesis missing.)